gradle.properties

```diff
@@ -1,4 +1,4 @@
-release_version=2.1.3
+release_version=2.1.4
```

Comment:
It should be 2.2.0, as it extends API, not fixes a bug nor makes internal change.